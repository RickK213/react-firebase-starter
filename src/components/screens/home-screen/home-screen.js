import React from 'react';
import { name, version } from '../../../../package.json';

export const HomeScreen = () => {
  const greeting = `Welcome to ${name} `;

  return (
    <div>
      <div
        style={{
          display: 'flex',
          justifyContent: 'space-between',
          marginTop: '20px'
        }}
      >
        <h2 style={{ marginTop: 0 }}>{greeting}</h2>
        {/* eslint-disable-next-line react/jsx-one-expression-per-line */}
        <code>v{version}</code>
      </div>
      <h4>
        <span role="img" aria-label="Under Construction">
          ⚠️
        </span>
        Under Construction
      </h4>
      <p>
        This is a starter application implementing React, Redux and Firebase.
      </p>
    </div>
  );
};
