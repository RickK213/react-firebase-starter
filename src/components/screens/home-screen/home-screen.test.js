/* global React, create */
import { HomeScreen } from './home-screen';

describe.skip('<HomeScreen />', () => {
  it('should render correctly', () => {
    const component = create(<HomeScreen />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
