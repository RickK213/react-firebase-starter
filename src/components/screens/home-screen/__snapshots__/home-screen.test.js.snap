// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeScreen /> should render correctly 1`] = `
<div>
  <div
    style={
      Object {
        "display": "flex",
        "justifyContent": "space-between",
        "marginTop": "20px",
      }
    }
  >
    <h2
      style={
        Object {
          "marginTop": 0,
        }
      }
    >
      Welcome to react-redux-firebase-starter 
    </h2>
    <code>
      Version 
      0.0.0
    </code>
  </div>
  <h4>
    <span
      aria-label="Under Construction"
      role="img"
    >
      ⚠️
    </span>
    Under Construction
  </h4>
  <p>
    This is a starter application implementing React, Redux and Firebase.
  </p>
</div>
`;
